python/tk_multi_loader/__init__.py

```diff
@@ -53,7 +53,6 @@ def show_dialog(app):
     # hide splash screen after loader UI show
     splash.finish(w.window())
 
-    # pop up help screen
+    # pop up welcome window
     if w.is_first_launch():
-        # wait a bit before show window
-        QtCore.QTimer.singleShot(1400, w.show_help_popup)
```

Comment:
Why do we remove the timer?

Reply:
It didn't seem to work with the timer, and I wasn't sure it was necessary. But I can try adding it back.

Reply:
After trying out the branch myself, I don't mind this to show the welcome message without a timer.